airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -324,3 +316,99 @@ def _set_name(self, name):
             return None
         validate_key(name, max_length=220)
         return re.sub(r'[^a-z0-9.-]+', '-', name.lower())
+
+    def create_new_pod_for_operator(self, labels, launcher) -> Tuple[State, k8s.V1Pod, Optional[str]]:
+        '''
```

Comment:
```suggestion
        """
```

To follow our convention